_test/ensure-stubs-compile.sh

```diff
@@ -8,10 +8,10 @@ if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
     grep "exercises/" |
     cut -d '/' -f -3 |
     sort -u |
-    awk -v repo=$repo '{print repo"/"$1}'
+    awk -v repo="$repo" '{print repo"/"$1}'
   )"
 else
-	changed_exercises=$repo/exercises/*/*
+	changed_exercises="$repo/exercises/*/*"
```

Comment:
```suggestion
	changed_exercises="$repo"/exercises/*/*
```

We want to keep those globs.